src/animations/backwardsCompatibleAnimWrapper.js

```diff
@@ -0,0 +1,96 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { default as Value } from '../core/AnimatedValue';
+import { evaluateOnce } from '../derived/evaluateOnce';
+
+function backwardsCompatibleInvoke(node, AnimationClass, value, config) {
+  let returnMethod;
+  const newValue = new Value(0);
+  const newClock = new Clock();
+  const currentState = AnimationClass.getDefaultState();
+  currentState.position = newValue;
+  let alwaysNode;
+  let isStarted = false;
+  let isDone = false;
```

Comment:
Do we really need 3 variables for keeping up with animation state? It seems like the only thing we want is to prevent `start` and `stop` being called twice.

Also note that since animation runs in native there is a chance that `isDone` may not yet been set to `true` even so the animation has ended in which case using it this way may not be 100% correct

Reply:
`isStarted` is need get info if is method start is triggered for 
```
      if (isStarted) {
        returnMethod && returnMethod({ finished: false });
        return;
      }
```
on start callback

`isDone` is needed not to start animation for the second time without callback and give proper warn

`wasStopped` is needed to give proper callback in `call`  in 48 line